src/SignalRServiceExtension/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class Constants
+    {
+        public const string AzureSignalRUserPrefix = "asrs.u.";
```

Comment:
Either make this a private member in `AzureSignalRClient` or move some other const string to this file together. It seems not necessary to put it in constants alone.